LOGGING/test/EventSourceListener.Tests/EventSourceTelemetryModuleTests.cs

```diff
@@ -294,7 +294,7 @@ public void ReportsAllProperties()
                 expected.Properties.Add("EventId", TestEventSource.ComplexEventId.ToString(InvariantCulture));
                 expected.Properties.Add("EventName", nameof(TestEventSource.ComplexEvent));
                 expected.Properties.Add("ActivityId", activityId.ToString());
-                expected.Properties.Add("Keywords", "0x8000F00000000001");
+                expected.Properties.Add("Keywords", "0x0000F00000000001");
```

Comment:
Do you know why this change was necessary?

Reply:
i changed once from 0000f to 8000f because tests were failing. after that, i reverted, so i did another commit with the original value. so 8000f to 0000f

Reply:
This caught my attention because all we changed here was the framework of the test project, but this suggests that there was a change in how the product was setting values on the telemetry.
I don't have the context on what this keyword is used for.
@cijothomas did you notice this change?

Reply:
the test was previously run only for 1.1. Is it failing for 1.1 now ? or failing for 2.1 and 3.1 only?

Reply:
locally its working for all, and, if i'm not mistaken, it was passing in server build as well

Reply:
Stepping back,
I see that in the CSPROJ we changed `netcoreapp1.0` to `netcoreapp1.1;netcoreapp2.1;netcoreapp3.1`

I spoke to Dmitry and he said that for EventKeywords, only the last few characters matter. He said the first character the object Type for this value.

@eddynaka can you test this locally with netcoreapp1.0?
If you can verify that netcore 1.0 passed with "0x0" then I think we can attribute this change to the changing frameworks and it's safe to merge!

Reply:
with 1.0 we need the 0x0 value. so, in develop, the test is passing.